src/Tribe/Assets.php

```diff
@@ -99,7 +99,11 @@ public function admin_enqueue_scripts() {
 				[ 'event-tickets-attendees-list-js', 'attendees-list.js', [ 'jquery' ] ],
 				[ 'event-tickets-admin-accordion-js', 'accordion.js', [] ],
 				[ 'event-tickets-admin-accordion-css', 'accordion.css', [] ],
-				[ 'event-tickets-admin-js', 'tickets.js', [ 'jquery-ui-datepicker', 'tribe-bumpdown', 'tribe-attrchange', 'tribe-moment', 'underscore', 'tribe-validation', 'event-tickets-admin-accordion-js', 'tribe-timepicker' ] ],
+				[ 'event-tickets-admin-js', 'tickets.js', [ 'jquery-ui-datepicker', 'tribe-bumpdown', 'tribe-attrchange', 'tribe-moment', 'underscore', 'tribe-validation', 'event-tickets-admin-accordion-js', 'tribe-timepicker' ], 'admin_enqueue_scripts',
```

Comment:
* :no_entry_sign: Array item not aligned correctly; expected 12 spaces but found 16
* :no_entry_sign: The first value in a multi-value array must be on a new line
* :no_entry_sign: The first value in a multi-value array must be on a new line
* :no_entry_sign: The first value in a multi-value array must be on a new line
* :no_entry_sign: The first value in a multi-value array must be on a new line
* :no_entry_sign: Each item in a multi-line array must be on a new line
* :no_entry_sign: Each item in a multi-line array must be on a new line
* :no_entry_sign: Each value in a multi-line array must be on a new line
* :no_entry_sign: Each item in a multi-line array must be on a new line
* :no_entry_sign: Each value in a multi-line array must be on a new line
* :no_entry_sign: Each item in a multi-line array must be on a new line
* :no_entry_sign: Each value in a multi-line array must be on a new line

<sub>hash: 3ecf9afdf4ff4e1474ed8a2d5071d39d</sub>